packages/suite/src/utils/suite/stake.ts

```diff
@@ -460,3 +472,75 @@ export const getStakeTxGasLimit = async ({
         };
     }
 };
+
+export const getUnstakingPeriodInDays = (validatorWithdrawTimeInSeconds?: number) => {
+    if (!validatorWithdrawTimeInSeconds) {
```

Comment:
this can be both `0` and `undefined` is it intended to pass this condition in both cases?

Reply:
17a49b68d6ccfd6fdac64509f574d96be9480573